cypress/pageObjects/settings/assertions.ts

```diff
@@ -0,0 +1,8 @@
+class SettingsPageAssertions {
+  checkHoemPageIsOpen() {
+    cy.url().should("equal", "http://demo.realworld.io/#/");
```

Comment:
after merging this PR https://github.com/AzizaAlSayed/cypress-test-repo/pull/12
you can use the checkUrlContainsValue assertion.